workspaces/ui-v2/src/components/navigation/NavButton.tsx

```diff
@@ -15,7 +15,7 @@ export function NavButton(props: NavButtonProps) {
   const { Icon, title, to } = props;
 
   return (
-    <Link className={classes.root} href={to}>
+    <Link className={classes.root} to={to}>
```

Comment:
Used the react-router link so we can use a react-router browser redirecet rather than a full page browser reload

Reply:
Wow, what a miss on my part 😅😭😬